capture.py

```diff
@@ -126,9 +136,10 @@ def capture(camera=None,
 
         with contextlib.nested(
              _maintain_camera(panel, camera),
-             _applied_viewport_options(viewport_options, panel),
```

Comment:
Purely reducing the complexity of the apply method. Could actually move this back to separate methods now that I think of it.